lib/api/bucketGet.js

```diff
@@ -63,9 +238,13 @@ export default function bucketGet(authInfo, request, log, callback) {
         delimiter: params.delimiter,
         marker: params.marker,
         prefix: params.prefix,
+
     };
+    if (versions) {
+        listParams.versions = versions;
+    }
```

Comment:
I don't think you use listParams.versions so don't need to do this. If you do need it, it would be clearer to just set listParams.versions to true.